pkg/block/s3/main_test.go

```diff
@@ -25,16 +25,16 @@ var (
 	pool     *dockertest.Pool
 )
 
-func newClient(port string) *minio.Client {
+func newClient(port string) (*minio.Client, error) {
 	creds := credentials.NewStaticV4(minioTestAccessKeyID, minioTestSecretAccessKey, "")
 
 	client, err := minio.New(fmt.Sprintf("%s:%s", minioTestEndpoint, port), &minio.Options{
 		Creds: creds,
 	})
 	if err != nil {
-		log.Fatalf("minio.New: %s", err)
+		return nil, err
 	}
-	return client
+	return client, nil
```

Comment:
no need to `if` as we do not wrap the error - just return what we get from `minio.New`